pyaedt/desktop.py

```diff
@@ -710,6 +710,63 @@ def __getitem__(self, project_design_name):
 
         return get_pyaedt_app(projectname, designname, self)
 
+    @pyaedt_function_handler()
+    def active_design(self, project_object, name=None, design_type=None):
+        """Get the active design.
+
+        Parameters
+        ----------
+        project_object :
+            AEDT project object.
+
+        name : str, optional
+            Name of the active design to make active.
```

Comment:
```suggestion
            Name of the design to make active.
```
Having active in this sentence twice is confusing.